services/web-app/components/catalog-search/catalog-search.js

```diff
@@ -40,15 +151,46 @@ const CatalogSearch = ({ onSearch }) => {
               size="lg"
             />
           </Column>
-          <Column className={styles.column} sm={2} md={4} lg={4}>
-            <MultiSelect
-              id="catalog-filter"
-              label="Filters"
-              items={items}
-              itemToString={(item) => (item ? item.text : '')}
-              initialSelectedItems={[items[0], items[1]]}
-              size="lg"
-            />
+          <Column sm={1} md={4} lg={4}>
+            <button
```

Comment:
What if all this logic was in a new `MultiSelect` component so its `handleSelect()`, `handleRemoveItem()`, `handleRemoveAll()` would be local to that component's source files and not part of the `CatalogSearch` component?

I don't know a good name... `MultiSelectFilter` or something?